apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -187,10 +187,11 @@ platform :ios do
 
       if (options[:ci])
         sh(
-          <<-SHELL
-           echo "IOS_BUILD_NUMBER=#{build_number+1}" >> "$GITHUB_ENV"
-          SHELL
-        )
+            <<-SHELL
+              new_build_number="IOS_BUILD_NUMBER=#{build_number+1}"
```

Comment:
nit: the variable contains the _build number assignment statement_, not just the build number. I'd choose a different variable name for that

Reply:
answered here https://github.com/LedgerHQ/ledger-live/pull/8339#discussion_r1837708153

Reply:
Hmm, I think one of us has misunderstood.

I'm saying `new_build_number` is not a build number, it's a build number assignment statement string

I suggest `new_build_number` ==> `new_build_number_assignment_cmd` or similar

Reply:
ah ok got you ! i've replaced it with `temp_ios_build_number_assignment`